src/redux/greeting/greetingSlice.js

```diff
@@ -0,0 +1,45 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+
+const initialState = {
+  greeting: [],
+  loading: false,
+  error: null,
+};
+
+export const fetchGreeting = createAsyncThunk(
+  'greeting/fetchGreeting',
+  async () => {
+    try {
+      const response = await fetch(`${process.env.REACT_APP_API_ENDPOINT}/greetings/random_greeting`, {
```

Comment:
**OPTIONAL** - Please add in your readme, reminding any user to change the URL to their localhost, this would help make your repo more professional